docs/components/collapse.md

```diff
@@ -0,0 +1,232 @@
+If no label is provided, only a caret will be displayed.
+
+<KCollapse trigger-alignment="leading">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
+</KCollapse>
+
+```html
+<KCollapse trigger-alignment="leading">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
+</KCollapse>
+```
+
+### v-model
+
+KCollapse can be controlled with `v-model`.
+
+<div class="mb-3">
+  <div>
+    <KLabel>Is Collapsed:&nbsp;</KLabel>
+    {{ myIsCollapsed }}
+  </div>
+  <KButton @click="myIsCollapsed = !myIsCollapsed">Another trigger</KButton>
+</div>
+<div>
+  <KCollapse trigger-label="View more info" v-model="myIsCollapsed">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
+  </KCollapse>
+</div>
+
+```html
+<div class="mb-3">
+  <div>
+    <KLabel>Is Collapsed:&nbsp;</KLabel>
+    {{ myIsCollapsed }}
+  </div>
+  <KButton @click="myIsCollapsed = !myIsCollapsed">Another trigger</KButton>
+</div>
+<div>
+  <KCollapse trigger-label="View more info" v-model="myIsCollapsed">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur
+  </KCollapse>
+</div>
+```
+
+### modelValue
+
+To set the default collapse state without binding to v-model you can use `modelValue`.
+
+<KCollapse title="Look Mah!" :model-value="false">
+  I am expanded by default
+</KCollapse>
+
+```html
+<KCollapse title="Look Mah!" :model-value="false">
+  I am expanded by default
+</KCollapse>
+```
+
+## Slots
+
+- `default` - Content to be hidden or shown when clicking the trigger
+- `visible-content` - Content displayed above the collapsible content that is always visible
+- `trigger-content` - Contents of the trigger; click handling is built-in
+- `trigger` - Completely control the trigger, including managing click events
```

Comment:
For `trigger` I think we need to document the `{ isCollapsed, toggle }` things that are available.

Reply:
![image](https://user-images.githubusercontent.com/67973710/195871876-1a8e0118-feb5-4545-9c73-7cd0572b8857.png)
 Updated